velox/vector/tests/DecodedVectorTest.cpp

```diff
@@ -443,14 +556,23 @@ TEST_F(DecodedVectorTest, dictionaryOverLazy) {
 
   // Ensure we set the baseVector_ in the DecodedVector, when decoding
   // DICT(LAZY) with loadLazy=false.
-  SelectivityVector selection(dictionarySize);
-  DecodedVector decoded(*dictionaryVector, selection, false);
-  auto base = decoded.base();
-  EXPECT_NE(base, nullptr);
-
-  // Ensure we don't load the lazy vector under dictionary.
-  auto lazyVector2 = dictionaryVector->valueVector()->as<LazyVector>();
-  EXPECT_FALSE(lazyVector2->isLoaded());
+  {
+    SelectivityVector selection(dictionarySize);
```

Comment:
Any particular reason you are not using the same pattern of 'check' lambda?

Reply:
probably it was the first one i changed before i saw the pattern

Reply:
i will change this one